docs/content/features/virtual-hosting.md

```diff
@@ -0,0 +1,29 @@
+# Virtual Hosting
+
+**SWS** provides rudimentary support for name-based [virtual hosting](https://en.wikipedia.org/wiki/Virtual_hosting#Name-based). This allows you to serve files from different root directories depending on the ["Host" header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/host) of the request, with all other settings staying the same.
+
+!!! warn "All other settings are the same!"
+    Each virtual host has to have all the same settings (aside from `root`). If using TLS, your certificates will have to cover all virtual host names as Subject Alternative Names (SANs). Also beware of other conflicting settings like redirects and rewrites. If you find yourself needing different settings for different virtual hosts, it is recommended to run multiple instances of SWS.
+
+Virtual hosting can be useful for serving more than one static website from the same SWS instance, if it's not otherwise feasible to run multiple instances of SWS. Browsers will automatically send a "Host" header which matches the hostname in the URL bar, which is how HTTP servers are able to tell which "virtual" host that the client is accessing.
```

Comment:
Despite Microsoft Word's insistence that phrases like "able to" should be "simplified" to "can", they don't actually mean the same thing exactly: https://english.stackexchange.com/a/196096 "Able to" exclusively refers to _ability_, but "can" has ambiguity since it could refer to either [_ability_ or _permission_](https://dictionary.cambridge.org/grammar/british-grammar/can). Neither one is more correct than the other.

"Which" and "that" in this context are widely used interchangeably: https://www.merriam-webster.com/grammar/when-to-use-that-and-which

The extra "that" here is "[not strictly wrong](https://academicguides.waldenu.edu/formandstyle/writing/scholarlyvoice/that)". I argue that using "that" in places like this is a stylistic choice.

These could be regional things, but this is just how I talk and write, which is the [ultimate grammatical argument](https://en.wikipedia.org/wiki/Linguistic_description) :)

Reply:
Fair enough :)